packages/i18n/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact i18n module, newest changes on the top.
 
+## [unreleased]
+
+### Fixed
+
+- `docs/index.md` broken link to i18n sample app repository
```

Comment:
We don't write changelog about fixing docs.

Reply:
OK.